platform/android/settings.gradle.kts

```diff
@@ -25,7 +25,7 @@ plugins {
 
 include(":MapLibreAndroid", ":MapLibreAndroidTestApp", ":MapLibreAndroidLint")
 
-rootProject.name = "MapLibre Native for Android"
```

Comment:
The spaces were causing problems with Dokka, so I searched if Gradle even supports spaces in project names and found this: https://discuss.gradle.org/t/space-character-in-build-name/24164

seems like spaces are discouraged (and Dokka errors out with them now)